packages/ui/src/modal/bid/TokenStatsHeader.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTokens, useCollections } from '../../hooks'
 import React, { FC } from 'react'
 import { styled } from '../../../stitches.config'
-import { Box, Text } from '../../primitives'
+import { Box, ChainIcon, Flex, Text } from '../../primitives'
```

Comment:
what's the difference between TokenStats and TokenStatsHeader? do they get used separately?

Reply:
Looks like TokenStatsHeader is used within TokenStats. These are components we've had for a while though. I think when we update the BidModal, we will get rid of these